### ISSUE TITLE
Feature flags

### ISSUE THREAD

**Issue by rozbb** [trailofbits/siderophile#7]
It would be helpful to be able to specify feature flags to compile the crate with when creating the callgraph

**Comment by disconnect3d** [comment]
Can't we do that through `Cargo.toml` of whatever crate we run Siderophile in? @rozbb

**Comment by rozbb** [comment]
Yes, you're right. It might be redundant to support both, but I think it'd be nice
